<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeWidget" name="windowListTreeWidget">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="renderHints">
         <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::NonCosmeticDefaultPen|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="nwiLabel">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Native window ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="nativeWindowIdLabel">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="wtLabel">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Window title:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="wbLabel">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Window bounds:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QFormLayout" name="formLayout_2">
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>12</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="leftLabel">
             <property name="text">
              <string>Left:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="boundsLeftSpinBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="topLabel">
             <property name="text">
              <string>Top:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="boundsTopSpinBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="widthLabel">
             <property name="text">
              <string>Width:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="boundsWidthSpinBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="heightLabel">
             <property name="text">
              <string>Height:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="boundsHeightSpinBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="windowTitleLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="windowLevelLabel">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Window level:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="windowLevelComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
